docs/1___overview.adoc

```diff
@@ -154,12 +154,12 @@ This major release of the SSP standard provides support for FMI 3.0, as well as
 Even though SSP 2.0 is a major version release, all SSP 1.0 conforming artifacts are considered to be fully conforming to SSP 2.0.
 SSP 2.0 conforming artifacts are only compliant to the SSP 1.0 standard if they use a 1.0 version number and only use features available in SSP 1.0.
 
-The following changes were performed as part of the 2.0.0 major release:
+An overview of the changes in the 2.0.0 major release are provided in the bulleted lists below. Many different advancements to the specification have been discussed; however, the SSP 2.0 specification has, in its current status, been deduced as a feasible least-common denominator for the exchange of simulation packages that is able to provide value for both end-users and tool suppliers.
 
 * Support for FMI 3.0
   ** Support for the new basic data types: Float32/64, and [U]Int8/16/32/64.
-  ** Support for the new scheduled execution implementation kind in the component implementation attribute.
-  ** Support for clocks as a new connector kind, and clocked variables/connectors.
+  ** Support for the new scheduled execution implementation kind in the component implementation attribute. Scheduled execution enables, for example, enriched event controlled execution of SSP components.
+  ** Clocked signals were introduced to FMI 3.0 for more exactly control timing of events and evaluation of model partitions across FMUs. SSP 2.0 introduces clocks as a new connector kind, and clocked variables/connectors.
```

Comment:
I don't think our changes section is the place to try to explain FMI 3.0 features - that is for the FMI standard and/or FMI Guides to do.